src/index.tsx

```diff
@@ -327,30 +420,62 @@ const styles = StyleSheet.create({
   flexOne: {
     flex: 1,
   },
-  flatList: {
+  flatListH: {
     flex: 1,
     flexDirection: isAndroidRTL ? 'row-reverse' : 'row',
   },
-  paginationContainer: {
+  flatListV: {
+    flex: 1,
+  },
+  paginationContainerH: {
     position: 'absolute',
     bottom: 16,
     left: 16,
     right: 16,
     justifyContent: 'center',
   },
-  paginationDots: {
+  paginationContainerV: {
+    position: 'absolute',
+    bottom: 16,
+    right: 8,
+    top: 16,
+    justifyContent: 'center',
+  },
+  paginationSafeAreaH: {
+
+  },
+  paginationSafeAreaV: {
+    position: 'absolute',
+    right: 8,
+    bottom: 16,
+    top: 16,
+    justifyContent: "center"
+  },
```

Comment:
Why is this style necessary?

Reply:
I have fixed this by using single class for both orientations